scripts/core/editor3/helpers/selection/blockKeys.ts

```diff
@@ -1,8 +1,12 @@
-export const getBlockKeys = (contentState, start, end) => [
-    ...contentState
+export interface IContentState {
```

Comment:
imo would be better to just use imported `ContentState` here instead of making things up